OpenRA.Game/Primitives/Color.cs

```diff
@@ -0,0 +1,361 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Globalization;
+using OpenRA.Scripting;
+
+namespace OpenRA.Primitives
+{
+	public struct Color : IScriptBindable
+	{
+		readonly long argb;
+
+		public static Color FromArgb(int red, int green, int blue)
```

Comment:
I commented about this above - these signatures are either copied directly from or modelled after `System.Drawing.Color` to keep compatibility.

If there is a consensus that we should aim for correctness instead then I can do that, but it will mean expanding the diff in this PR even further as all the places that use these will obviously need to change too.